recipe/meta.yaml

```diff
@@ -11,30 +14,33 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
   script:
     - {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
 
 requirements:
   host:
-    - python >=3.8
+    - python {{ python_min }}
     - pip
     - setuptools
     - hatchling >=1.5.0
     - hatch-nodejs-version >=0.3.2
     - hatch-jupyter-builder >=0.5
-    - pynvml
-    - psutil
     - nodejs
     - jupyter-packaging
     - jupyterlab >=4
```

Comment:
Please double check and let me know if this still looks accurate or needs updates

Reply:
I think this should be fine. Most of the recent changes were on the front end.